examples/jsm/tsl/display/SSRNode.js

```diff
@@ -198,6 +254,17 @@ class SSRNode extends TempNode {
 		 */
 		this._textureNode = passTexture( this, this._ssrRenderTarget.texture );
 
+		const mips = this._blurRenderTarget.texture.mipmaps.length - 1;
+		const lod = this.metalnessRoughnessNode.g.mul( mips ).clamp( 0, mips );
```

Comment:
@sunag I was initially wondering why this code did not always select the full range of mips until I found out that the TSL object `roughness` not just represents the `roughness` material value combined with the value from `roughnessMap` but also the geometry roughness which is added via below code.

https://github.com/mrdoob/three.js/blob/1cda8f3b681651fb03142821d9a7b9721ad76659/src/materials/nodes/MeshStandardNodeMaterial.js#L159

I believe this makes the SSR more physically plausible. However, it's important to know that otherwise it's irritating when playing with the roughness slider in the demo (e.g. you can't force `0` roughness for all fragments).

Reply:
Have you tried with `materialRoughness`?